pages/science/flames.rst

```diff
@@ -125,6 +125,40 @@ where :math:`D_{ki}` is the multicomponent diffusion coefficient and
 :math:`D_k^T` is the Soret diffusion coefficient (used only if calculation of
 this term is specifically enabled).
 
+The Drift-Diffusion Model
+^^^^^^^^^^^^^^^^^^^^^^^^^
+
+To account for the transport of charged species in a flame, the drift term is added to the
+diffusive fluxes of the mixture-average formulation according to [Ped1993]_,
```

Comment:
This is only done if the `IonFlow` domain is being used, right? I think that should be clarified.

Reply:
ok